internal/message_server.py

```diff
@@ -72,9 +72,9 @@ def get(self):
             response += "</style>\n"
             response += "</head><body><div id='wptorange'></div>\n"
             if MESSAGE_SERVER.config is not None:
-                import cgi
+                import html
```

Comment:
You either need to special-case this for python >= 3.2 or we need to make the call the python 2.7 is no longer supported (in which case the change is fine).

My vote is to stop supporting 2.7 and clean up the code. @tkadlec any thoughts?

Reply:
I agree. 3.2 has been out for 11 years now. :) Think it's safe to drop 2.7 support.